### PR TITLE
docs(web-modeler): improve documentation on Web Modeler Self-Managed

### PR DESCRIPTION
**Pull request by jfriedenstab**
## What is the purpose of the change
Address review comments from @christian-konrad for #1356 (that has already been merged).

## Are there related marketing activities
see #1356

## When should this change go live?
see #1356

## PR Checklist

- [x] My changes apply to an already released version, and I have added them to the relevant `/versioned_docs` directory, or they are not for an already released version.
- [x] My changes apply to future versions, and I have added them to the main `/docs` directory, or they are not for future versions.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: src/css/custom.css [hunk 1]
```diff
@@ -195,6 +195,11 @@ svg.implemented:hover [stroke="#333"] {
   margin-bottom: 1em;
 }
 
+.badge--beta {
+  background-color: #26d07c;
```
Comment (jfriedenstab) [inline src/css/custom.css:199]: FYI: Feel free to choose a different color here.

**Comment by akeller** [comment]
Feel free to merge this with approvals from the current reviewers. We won't be able to accommodate extensive feedback on this PR since we are looking to work on some docs infrastructure that will effectively freeze our docs in the US early afternoon (about 1 hour away). 

If no one is available to review, I will review and merge when the 2 other PRs I'm watching are on `main`.

File: src/css/custom.css (hunk 1 above)
Comment (christian-konrad) [reply src/css/custom.css:199]: This one works well I guess

**Review by christian-konrad — APPROVED** [review]
All good, I just added another beta badge to `about-self-managed.md`

**Comment by akeller** [comment]
Sorry about the merge conflicts! 😬 I can handle them when I'm done with my lunch.

**Review by akeller — APPROVED** [review]

**Comment by akeller** [comment]
My brain is a little mushy but I think we are looking for `../../reference/early-access#beta` here 😅 

Testing the build locally, and if it's good I'll commit.

**Review by akeller — APPROVED** [review]

**Comment by christian-konrad** [comment]
Thanks @akeller for having a last view before the release!

**Pull request merged**